Jenkinsfile

```diff
@@ -9,7 +9,7 @@ pipeline {
     BASE_DIR = "src/github.com/elastic/${env.REPO}"
     NOTIFY_TO = credentials('notify-to')
     JOB_GCS_BUCKET = credentials('gcs-bucket')
-    PIPELINE_LOG_LEVEL='INFO'
+    PIPELINE_LOG_LEVEL = 'DEBUG'
```

Comment:
Is this still required?

Reply:
Evil GH issues... for some reason it did not push my changes

```
apm-agent-rum-js git:(feature/report-bundlesize) git --no-pager log -1       
commit a990a52150719ba7b1b4d6c64e00249e0e58dfa9 (HEAD -> feature/report-bundlesize, origin/feature/report-bundlesize)
Author: Victor Martinez <VictorMartinezRubio@gmail.com>
Date:   Mon Jun 29 10:59:06 2020 +0100

    ci: use current branch and change comment
```

Reply:
I've just had to amend my last commit and force push